cmd/abapLandscapePortalUpdateAddOnProduct_test.go

```diff
@@ -0,0 +1,816 @@
+package cmd
+
```

Comment:
Could you add an "end-to-end" unit test to cover all steps (of course with mocked API calls)?